packages/inference/src/tasks/audio/audioClassification.ts

```diff
@@ -16,10 +16,12 @@ export async function audioClassification(
 	options?: Options
 ): Promise<AudioClassificationOutput> {
 	const payload = preparePayload(args);
-	const res = await request<AudioClassificationOutput>(payload, {
-		...options,
-		task: "audio-classification",
-	});
+	const res = (
+		await innerRequest<AudioClassificationOutput>(payload, {
+			...options,
+			task: "audio-classification",
+		})
+	).data;
```

Comment:
This syntax feels more "idiomatic"
```suggestion
	const { data: res } = await innerRequest<AudioClassificationOutput>(payload, {
		...options,
		task: "audio-classification",
	});
```

Reply:
Applied this suggestion in all tasks here [68fda16](https://github.com/huggingface/huggingface.js/pull/1314/commits/68fda16ccffdf5bb91a5cf8800a05a026e90dde0)